waku/factory/external_config.nim

```diff
@@ -297,13 +297,6 @@ type WakuNodeConf* = object
       name: "keep-alive"
     .}: bool
 
-    topics* {.
-      desc:
-        "Default topic to subscribe to. Argument may be repeated. Deprecated! Please use pubsub-topic and/or content-topic instead.",
-      defaultValue: @["/waku/2/default-waku/proto"],
-      name: "topic"
-    .}: seq[string]
```

Comment:
I would love to remove it but wasn't there someone still using this?

Reply:
the deprecation was announced many months ago.